examples/TF_1_x/object_detection_sample.py

```diff
@@ -164,10 +156,18 @@ def non_max_suppression(predictions_with_boxes,
     return result
 
 
+def load_labels(label_file):
+    label = []
+    proto_as_ascii_lines = tf.io.gfile.GFile(label_file).readlines()
+    for l in proto_as_ascii_lines:
+        label.append(l.rstrip())
+    return label
+
+
 if __name__ == "__main__":
-    input_file = "examples/data/people-detection.mp4"
-    model_file = "examples/data/yolo_v3_darknet.pb"
-    label_file = "examples/data/coco.names"
+    input_file = "data/grace_hopper.jpg"
```

Comment:
not consistent with README instructions. No additional data folder needed in TF_1_x folder.